bindings/c/test/fdb_c_client_config_tests.py

```diff
@@ -568,14 +568,14 @@ def test_external_client(self):
         test.api_version = api_version_from_str(PREV_RELEASE_VERSION)
         test.exec()
         test.check_initialization_state("created")
-        test.check_healthy_status(False)
+        test.check_healthy_status(True)
```

Comment:
Hi @sfc-gh-vgasiunas , can you take a look at the change I made in this file in order to make the test passing? I tried my best to understand the test, but I may still miss something. Thanks!

The reason for health to be false in 7.3 is because in 7.1, there isn't client get status functionality. But since 7.3 has, it becomes true.

Another way to make this test pass is to change `PREV_RELEASE_VERSION` to `PREV2_RELEASE_VERSION`

Reply:
Yes, update to True makes sense